apps/calendar/test/marionette/week_view_test.js

```diff
@@ -313,12 +320,31 @@ marionette('week view', function() {
       assert.equal(editEvent.startTime, pad(time) + ':00:00', 'startTime');
       assert.equal(editEvent.endTime, pad(time + 1) + ':00:00', 'endTime');
     }
+  });
 
-    function isShowingAddEventLink() {
-      return week.element.scriptWith(function(el) {
-        return !!el.querySelector('.md__add-event');
-      });
-    }
+  suite('event retargeting tests', function() {
+    test('When tapping around an event we can create another event',
+    function() {
+      week.waitForHourScrollEnd();
+
+      week.waitForNoScroll();
```

Comment:
Waiting twice for the scroll to be ended looks odd. I wonder if we could change `_waitForScrollEnd()` by `waitForNoScroll()`. I'll try that